packages/http-response-serializer/index.js

```diff
@@ -9,7 +9,10 @@ const defaults = {
 const httpResponseSerializerMiddleware = (opts = {}) => {
   const { serializers, defaultContentType } = { ...defaults, ...opts }
   const httpResponseSerializerMiddlewareAfter = async (request) => {
-    normalizeHttpResponse(request)
+    request.response = normalizeHttpResponse(request)
+
+    request.event ??= {}
+    request.event.headers ??= {}
```

Comment:
Event normalization should be in https://github.com/middyjs/middy/blob/main/packages/http-event-normalizer/index.js.
However,
1. How are you triggering a lambda without an event?
2. How are you triggering a lambda without headers in an HTTP event?

Reply:
Good question, I'm incrementally making my way through the cloudwatch logs to try and figure that out. I believe the inner code is throwing an error, returning something bad, then the response serialiser is trying to handle it but then throwing an error too, so I can't actually see the inner error right now. 

So while clearly I need to fix the inner code I feel like it makes sense to make the serialiser a bit more fault tolerant so the inner error can be seen.

```
ERROR	Invoke Error 	
{
    "errorType": "TypeError",
    "errorMessage": "Cannot read properties of undefined (reading 'Accept')",
    "stack": [
        "TypeError: Cannot read properties of undefined (reading 'Accept')",
        "    at httpResponseSerializerMiddlewareAfter (/var/task/handler.js:101142:50)",
        "    at runMiddlewares (/var/task/handler.js:97057:23)",
        "    at async runRequest (/var/task/handler.js:97035:7)"
    ]
}
```

I've moved the code as suggested and gone the route of using a null conditional instead.

Reply:
I found the issue this morning and it was relating to the way the data was being formatted when being passed into an integration test. It probably would've been easier to see had this outer error not occurred.

Reply:
After some refection, I opted to make a different change that had the same affect that was more consistent with other middlewares. See 4.6.6